vagrant/README.md

```diff
@@ -18,16 +18,35 @@ Currently it uses Kubernetes v1.6.1
 ## Features
 
 **CACHING:** This vagrant/ansible environment allows for caching of the yum
-cache and Docker images. This allows you to reuse those caches on subsequent
-provisioning of the VMs, and it is intended to help situations where one is
-developing in an environment where they would rather not have to redownload
-many megabytes repeatedly, e.g. hotel WiFi. :)  It stores both as `tgz` files
-in your `VAGRANT_HOME` directory, `~/.vagrant.d` by default. To enable this,
-either specify `VAGRANT_CACHE=1` on the command line or change the `CACHE`
-variable near the top of the Vagrantfile from `false` to `true`. **NOTE:** This
-will enable use of a Docker cache, but you must run `./docker_gen_cache.sh`
-from the `vagrant/` directory to initialize the cache. This is because
-creating the cache can take some time. It is recommended you pull any
-additional images you use to the master node, as this cache will be used by all
-nodes. For convenience, the GlusterFS and heketi images are automatically
-pulled to the master node.
+cache. This allows you to reuse those caches on subsequent provisioning of the
+VMs, and it is intended to help situations where one is developing in an
+environment where they would rather not have to redownload many megabytes
+repeatedly, e.g. hotel WiFi. :)  It stores both as `tgz` files in your
+`VAGRANT_HOME` directory, `~/.vagrant.d` by default. To enable this, either
+specify `VAGRANT_CACHE=1` on the command line or change the `CACHE` variable
+near the top of the Vagrantfile from `false` to `true`.
+
+**CUSTOM REGISTRY:** Similar to the caching feature, this environment supports
+interaction with a custom Docker registry. The idea is that a registry would
+be running in a local environment somewhere that could be used as a primary
+source for pulling container images into the VMs. Just specify the variable
+`custom_registry` in the `global_vars.yml` file to configure it. The following
```

Comment:
Since global_vars.yml is a checked-in file, I was thinking whether there would be a possibility to get the url from a separate file (not checked in) or from an env var, and only use the custom registry if that file/variable contains someting. ???